openapi/trireg2admin.yaml

```diff
@@ -0,0 +1,42 @@
+openapi: "3.0.3"
+info:
+  title: "TriReg2 Admin API"
+  description: "An OpenAPI specification for time registrations to be used by an administrator"
+  version: "0.1.2"
```

Comment:
Maybe a more significant bump in version number(s) due to significant breaking changes?

Reply:
good point. will change to v0.2.0